react-components/src/components/Architecture/DomainObjectPanel.tsx

```diff
@@ -20,44 +17,33 @@ import { type TranslateDelegate } from '../../architecture/base/utilities/Transl
 import { type UnitSystem } from '../../architecture/base/renderTarget/UnitSystem';
 import { IconComponent } from './Factories/IconFactory';
 import { type DomainObject } from '../../architecture';
+import { useSignalValue } from '@cognite/signals/react';
 
 const TEXT_SIZE = 'x-small';
 const HEADER_SIZE = 'medium';
 
 export const DomainObjectPanel = (): ReactElement => {
-  const [currentDomainObjectInfo, setCurrentDomainObjectInfo] = useState<
-    DomainObjectInfo | undefined
-  >();
-
-  const domainObject = currentDomainObjectInfo?.domainObject;
+  useSignalValue(DomainObjectPanelUpdater.update);
```

Comment:
Is this something we will need in the future? I would expect the UI to update if any of the things it uses changes without having to listen to a generic update signal.